custom_model_runner/datarobot_drum/drum/model_adapter.py

```diff
@@ -303,6 +303,7 @@ def transform(self, model=None, **kwargs):
         pd.DataFrame
         """
         input_binary_data = kwargs.get(StructuredDtoKeys.BINARY_DATA)
+        target_binary_data = kwargs.get(StructuredDtoKeys.TARGET_BINARY_DATA)
```

Comment:
why is it binary data instead of just data?

Reply:
@yakov-g changed some stuff....but now I need to go back and make sure it makes sense to follow suit when passing target

Reply:
@4dahalibut just to show this data is bytes